internal/toggles/toggle.go

```diff
@@ -1,5 +1,13 @@
 package toggles
 
+// Features : A feature toggle is a technique that attempts to provide an alternative to maintaining
```

Comment:
This seems to come from Unleash. This is too generic.

Reply:
"Features is an interface which allows you to enable specific behaviour for a given user. In particular, it controls whether Jenkins idling is enabled for a specific user.